src/CodeGenerators/SettingsGen/Models/Attributes/IgnoreAttribute.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+
+namespace Microsoft.Omex.CodeGenerators.SettingsGen.Models.Attributes
+{
+	/// <summary>
+	/// Ignore attribute which indicates the property should not be generated to settings file or used in validation
+	/// </summary>
+	[AttributeUsage(AttributeTargets.Property)]
+	public sealed class IgnoreAttribute : Attribute
```

Comment:
Could it be that we would face with other attributes that have the same name?

Reply:
So there is a risk of that but this attribute will only be looked for when it is on the property of class which also has the Attribute section on it so the risk of overlap should be 0